index.js

```diff
@@ -96,6 +96,11 @@ module.exports = {
         */
         'no-console': 'error',
 
+        /*
```

Comment:
I'm not in-favor of disabling this rule. I think it makes sense to discourage the use because it is a relatively under used feature (you found relatively few instances) and anecdotally I find code that is refactored to not use continue statements easier to read or equally readable.

I think there are cases, like streaming parsing where the algorithm lends naturally to using continue or usage in hot loops but I think it should be generally discouraged. I agree that developers can generally use it properly but I think it adds cognitive overhead in review and refactor because it's used infrequently and I would generally discourage use.

Reply:
This is a commonly used feature across many languages. It's very readable, easily learned, and requires minimal cognitive overhead.

Reply:
I also find myself having to think a little more when I see `continue` in code and generally prefer to refactor to avoid it. But I lean towards allowing it since:
1. our C# conventions don't disallow it
2. there are cases where it's perfectly understandable and even preferable, like when used at the start of a block to avoid extra nesting
```ts
// with continue at the beginning of a block: easy to understand and avoids 1 level of nesting in the main section of code
for (const val in array) {
    if (shouldSkip(val)) {
       continue;
    }
    // a bunch more code that has its own nesting
}

// without continue, extra nesting for the main section of code
for (const val in array) {
    if (!shouldSkip(val)) {
        // a bunch more code that has its own nesting
    } 
}
```

*Proposal:* allow `continue` but update the comment to discourage poor usage
```
/* `continue` statements can improve readability but care should be taken to avoid confusing control flow. They should generally be the only statement within an `if` block and should never use labels. */
'no-continue': 'off',
```

Reply:
We agreed in person to go with this proposal. @TrevorKarjanis can update with the proposed comment and complete the PR

Reply:
Updated wit the proposed comment 👍